src/components/searchBar.js

```diff
@@ -52,54 +70,112 @@ export const SearchBar = (props) => {
     inputRef.current.focus();
   }
 
+  function handleRenderSuggestion(item) {
+    return (
+      <TableRow key={item.symbol} onClick={() => handleSearch(item.symbol)}>
+        <TableColumn>
+          <Text color="darkblue" size="medium" mr="10px">
+            {item.symbol}
+          </Text>
+        </TableColumn>
+        <TableColumn>
+          <Text
+            variant="primary"
+            size="medium"
+            mr="16px"
+            display="inline-block"
+          >
+            {item.securityName}
+          </Text>
+          <Text
+            className="suggestionExchange"
+            variant="primary"
+            display="inline-block"
+            padding="2px 8px"
+          >
+            {item.exchange}
+          </Text>
+        </TableColumn>
+      </TableRow>
+    );
+  }
+
   useEffect(() => {
     if (isInputFocused) {
       setShowCompanyText(false);
-    } else if (currentText === '') {
+    } else if (currentText === "") {
       setShowCompanyText(true);
     }
-  }, [isInputFocused, currentText])
+  }, [isInputFocused, currentText]);
 
   return (
     <DisplayWrapper {...props}>
-      <form className="searchForm" onSubmit={handleSubmit}>
-      <Icon
-        className="searchIcon"
-        path={mdiMagnify}
-        size={1.5}
-        color="#5496ff"
-      />
-      <input
-        ref={inputRef}
-        className="searchBar"
-        onBlur={onLeaveInputFocus}
-        onFocus={onInputFocus}
-        value={currentText}
-        onChange={handleType}
-      />
-      {showCompanyText && (
-        <div className="companyText" onClick={onCompanyTextClick}>
-          <Text variant="primary" fontWeight="lighter" size="large" display="inline-block">
-            {companyName}
-          </Text>{" "}
-          <Text display={["none", "none", "inline-block"]} variant="secondary" fontWeight="lighter" size="large" >
-            {symbolText}
-          </Text>
-        </div>
-      )}
-
-      <div>
-        {showBadInputError && (
-          <Text mt="8px" variant="error">Not a valid input, searches should contain only letters</Text>
+      <form
+        className="searchForm"
+        onSubmit={(e) => {
+          e.preventDefault();
+          handleSearch(currentText);
+        }}
+      >
+        <Icon
+          className="searchIcon"
+          path={mdiMagnify}
+          size={1.5}
+          color="#5496ff"
+        />
+        <input
+          ref={inputRef}
+          className="searchBar"
+          onBlur={onLeaveInputFocus}
+          onFocus={onInputFocus}
+          value={currentText}
+          onChange={handleType}
+        />
+        {symbolSuggestions.length > 0 && (
```

Comment:
Sorry, there is a little bit of prettier styling fixes in this PR from previous work. This part is the main thing that changed in the JSX, adding the table.